src/content/Angular-CSharp/ClientApp/proxy.conf.js

```diff
@@ -17,7 +17,10 @@ const PROXY_CONFIG = [
 //#endif
    ],
     target: target,
-    secure: false
+    secure: false,
+    headers: {
+      Connection: 'Keep-Alive'
```

Comment:
This should only make a difference to the server if HTTP/1.0 (or HTTP/0.9) requests are proxied. Or if this is overriding an `Connection: close` header. Do we know what's actually happening? Do we have a network trace of the failed proxy attempt? @Tratcher 

And nit: `keep-alive` is normally lower case when it's part of the `Connection` header, but it is case-insensitive, so it doesn't really matter.